noir-projects/noir-protocol-circuits/crates/public-kernel-lib/src/common.nr

```diff
@@ -111,18 +111,18 @@ fn perform_static_call_checks(public_call: PublicCallData) {
     if public_inputs.call_context.is_static_call {
         // No state changes are allowed for static calls:
         let new_note_hashes_length = array_length(public_inputs.new_note_hashes);
-        assert(new_note_hashes_length == 0, "new_note_hashes must be empty for static calls");
+        assert(new_note_hashes_length == 0, "new_note_hashes must be default for static calls");
```

Comment:
I think the error messages are way more descriptive with the word `empty` in these cases